bot/moderation/moderation.py

```diff
@@ -15,49 +16,77 @@ def __init__(self, bot):
-        await ctx.author.send("Deine Nachricht wurde erfolgreich an die Moderatoren weitergeleitet!\n"
-                              "__Hier deine Bestätigung:__", embed=embed_confirmation)
+        if ctx.invoked_subcommand is None:
+            msg_content = ctx.message.content
+            msg_attachments = ctx.message.attachments
+            msg_author_name = str(ctx.message.author)
+            msg_timestamp = ctx.message.created_at
+            await ctx.message.delete()
+
+            ch_modmail = ctx.guild.get_channel(constants.CHANNEL_ID_MODMAIL)
+            msg_content = msg_content[len(ctx.prefix + ctx.command.name):]
+
+            embed = discord.Embed(title="Status: Offen", color=constants.EMBED_COLOR_MODMAIL_OPEN,
+                                  timestamp=datetime.utcnow(), description=msg_content)
+            embed.set_author(name=ctx.author.name + "#" + ctx.author.discriminator, icon_url=ctx.author.avatar_url)
+            embed.set_footer(text="Erhalten am")
+
+            msg_modmail = await ch_modmail.send(embed=embed, files=msg_attachments)
+            self._db_connector.add_modmail(msg_modmail.id, msg_author_name, msg_timestamp)
+            await msg_modmail.add_reaction(constants.EMOJI_MODMAIL_DONE)
+            await msg_modmail.add_reaction(constants.EMOJI_MODMAIL_ASSIGN)
+
+            embed_confirmation = embed.to_dict()
+            embed_confirmation["title"] = "Deine Nachricht:"
+            embed_confirmation["color"] = constants.EMBED_COLOR_INFO
+            embed_confirmation = discord.Embed.from_dict(embed_confirmation)
+            await ctx.author.send("Deine Nachricht wurde erfolgreich an die Moderatoren weitergeleitet!\n"
+                                  "__Hier deine Bestätigung:__", embed=embed_confirmation)
+
+    @modmail.command(name='get')
+    @commands.has_role(constants.ROLE_ID_MODERATOR)
+    async def get_modmail_with_status(self, ctx: discord.ext.commands.Context, *, status: str):
+        """Command Handler for the modmail subcommand `get`.
+
+        Allows moderators to generate an embeded message (Embed) in the modmail channel which contains a list of all
+        modmails with the specified status. Each list element is a hyperlink which, if clicked, brings you to the
+        corresponding modmail message.
+
+        Args:
+            ctx (discord.ext.commands.Context): The context in which the command was called.
+            status (str): The status specified by the moderator.
+        """
+        if ctx.channel.id == constants.CHANNEL_ID_MODMAIL:
+            try:
+                enum_status = ModmailStatus[status.upper()]
+
+                ch_modmail = ctx.guild.get_channel(constants.CHANNEL_ID_MODMAIL)
+                modmail = self._db_connector.get_all_modmail_with_status(enum_status)
+
+                embed = _modmail_create_list_embed(enum_status, modmail)
+                await ch_modmail.send(embed=embed)
+            except (KeyError, ValueError):
```

Comment:
Remove KeyError if it does not happen.

Reply:
A KeyError gets raised if there isn't a corresponding Enum constant for the specified value.
A ValueError gets raised by the bot itself if someone tries to use an unsupported status (e.g: `Closed`).

Maybe I will implement messages for the status `Closed` too but right now, everything works as expected.